sql/hive/src/test/scala/org/apache/spark/sql/hive/InsertSuite.scala

```diff
@@ -728,4 +732,254 @@ class InsertSuite extends QueryTest with TestHiveSingleton with BeforeAndAfter
       assert(e.contains("mismatched input 'ROW'"))
     }
   }
+
+  private def getConvertMetastoreConfName(format: String): String = format match {
+    case "parquet" => "spark.sql.hive.convertMetastoreParquet"
+    case "orc" => "spark.sql.hive.convertMetastoreOrc"
```

Comment:
Could you use keys?
- `HiveUtils.CONVERT_METASTORE_PARQUET.key` instead of "spark.sql.hive.convertMetastoreParquet"
- `HiveUtils.CONVERT_METASTORE_ORC.key` instead of "spark.sql.hive.convertMetastoreOrc"